testsuite/__init__.py

```diff
@@ -0,0 +1,111 @@
+# pylint: disable=missing-docstring
+from flask import Flask, render_template, request
+app = Flask(__name__)  # pylint: disable=invalid-name
+
+
+@app.route("/")
+def index():
+    return render_template('index.html')
+
+
+@app.route('/tests.json', methods=['POST'])
+def tests():
+    from behave.configuration import Configuration
```

Comment:
I'm not familiar with this style of importing from within a function — is there a reason it's desirable? http://legacy.python.org/dev/peps/pep-0008/#imports and http://stackoverflow.com/questions/1024049/is-it-pythonic-to-import-inside-functions seem down on it, but I'm not well-versed on this topic.